Code/Tools/AssetProcessor/native/ui/GoToButtonDelegate.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "GoToButtonDelegate.h"
+#include <qpainter.h>
+#include <qevent.h>
+
+namespace AssetProcessor
+{
+    GoToButtonDelegate::GoToButtonDelegate(QObject* parent)
+        : QStyledItemDelegate(parent)
+        , m_icon(":/AssetProcessor_goto.svg")
+        , m_hoverIcon(":/AssetProcessor_goto_hover.svg")
+    {
+
+    }
+
+    void GoToButtonDelegate::paint(QPainter* painter, const QStyleOptionViewItem& option, const QModelIndex& index) const
+    {
+        QStyledItemDelegate::paint(painter, option, index);
+
+        if (index.data().canConvert<GoToButtonData>())
+        {
+            if (option.state & QStyle::State_MouseOver)
+            {
+
+                m_hoverIcon.paint(painter, option.rect);
+            }
+            else
+            {
+                m_icon.paint(painter, option.rect);
+            }
```

Comment:
```suggestion
            if (option.state & QStyle::State_MouseOver)
            {
                m_hoverIcon.paint(painter, option.rect);
            }
            else
            {
                m_icon.paint(painter, option.rect);
            }
```
Could you potentially scale the icons down a bit to give them padding so they aren't so large here?  You may also need to size the icon in the constructor with:
```
m_icon= QIcon(":/AssetProcessor_goto.svg").pixmap(width, height);
m_hoverIcon= QIcon(":/AssetProcessor_goto_hover.svg").pixmap(width, height);
```